src/utils/config.ts

```diff
@@ -337,6 +337,7 @@ export const APP_ITEMS_PREFIX = '/app-items';
 export const THUMBNAILS_ROUTE_PREFIX = '/thumbnails';
 
 export const MEMBER_PROFILE_ROUTE_PREFIX = '/profile';
+export const MEMBER_EXPORT_DATA_ROUTE_PREFIX = '/member-data-export';
```

Comment:
I think the path already has member, right?  `members/member-data-export`?  Maybe it can be simply `members/data-export`?